src/daemon/rpmostreed-sysroot.c

```diff
@@ -432,6 +432,36 @@ reset_config_properties (RpmostreedSysroot  *self,
   return TRUE;
 }
 
+static gboolean
+reload_sysroot_and_os (RpmostreedSysroot *self,
+                       GError           **error)
+{
+  gboolean sysroot_changed;
+  if (!sysroot_reload (self, &sysroot_changed, error))
+    return FALSE;
+
+  /* always send an UPDATED signal to also force OS interfaces to reload */
```

Comment:
If we're doing this then shouldn't the method be called
`reload_sysroot_force_os_reload()` or something?

Reply:
Fixup for that! :arrow_down: